frontend/src/metabase/common/hooks/use-action-list-query/use-action-list-query.ts

```diff
@@ -0,0 +1,19 @@
+import { WritebackAction, WritebackActionListQuery } from "metabase-types/api";
+import {
+  useEntityListQuery,
+  UseEntityListQueryProps,
+  UseEntityListQueryResult,
+} from "metabase/common/hooks/use-entity-list-query";
+import Actions from "metabase/entities/actions";
+
+export const useActionListQuery = (
+  props: UseEntityListQueryProps<WritebackActionListQuery> = {},
+): UseEntityListQueryResult<WritebackAction> => {
```

Comment:
I don't like the `Action` vs `WritebackAction` inconsistency in the naming, but:
- it's `entities/actions`, not `entities/writeback-actions`
- renaming `WritebackAction` is a material for another day

Reply:
Agree, `WritebackAction` is used everywhere atm